homeassistant/components/kulersky/light.py

```diff
@@ -67,11 +62,6 @@ async def discover(*args):
     async_track_time_interval(hass, discover, DISCOVERY_INTERVAL)
```

Comment:
We should stop this subscription too on entry unload. Save the return value in hass.data so we can call it on unload and Home Assistant stop event.

Reply:
Thanks! I pushed up this fix for Zerproc as well.

Reply:
The change looks good, but normally we don't mix changes for different integrations in the same PR. Please update the PR title and description.